nengo_bones/version.py

```diff
@@ -22,3 +22,5 @@
     v=".".join(str(v) for v in version_info),
     dev=(".dev%d" % dev) if dev is not None else "",
 )
+
+copyright = "2018-2021 Applied Brain Research"
```

Comment:
Did a quick google of how these should be formatted, and how other Python projects format it and it seems like the consensus is `"Copyright (c) 2018-2021 Applied Brain Research"` so I'm going to use that.